### PR TITLE
HCBS - Admin View Page

### PR DESCRIPTION
**Pull request by rocio-desantiago**
### Description

A non-state user should see the same kind of experience as MCR/MFP when they login, and are able to select a state and report (QM) to link to the submissions view.
- Without completely following the same format of creating a <`Form`> and <`Dropdown`> component amongst several other `Types` involved in making this information appear on the other apps, we have a basic proof of concept here to include the same information until we decide if we want to continue with the same patterns.
- There were some discrepancies between MCR/MFP that were hashed out. 
- In addition to this work, to get MCR on the same page as MFP ([PR #11896](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/11896#event-14858117149))
     - the title was updated to "View State/Territory Reports"
     - the color scheme for disabled button confirmed that opacity should be 1 vs the lighter gray that was previously showing
See exhibit A:
admin pages compared

### Related ticket(s)
CMDCT-4004

---
### How to test

**Deploy Link:** https://d1p9pr8qngia3c.cloudfront.net/

- When you sign in as an admin, you should see the page to select a state and report.
- Select a state and a report
- When you click **Go to Report Dashboard**, the page will redirect to dashboard page with the URL ending in "/report/QM/AS"
select state as an admin

### Important updates

---
### Author checklist

- [x] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by BearHanded — APPROVED** [review]

File: services/ui-src/src/components/app/AppRoutes.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const AppRoutes = () => {
         <Route path="/" element={<HomePage />} />
         <Route path="/profile" element={<ProfilePage />} />
         <Route path="/help" element={<HelpPage />} />
-        <Route path="/dashboard" element={<DashboardPage />} />
```
Comment (ailZhou) [inline services/ui-src/src/components/app/AppRoutes.tsx:14]: i think something is missing, as if you do this and log in as a stateuser, we lost the dashboard page for the state.user

https://github.com/user-attachments/assets/a7cdac03-c5c8-43c8-a87a-29f343204de3

Comment (ailZhou) [reply services/ui-src/src/components/app/AppRoutes.tsx:14]: actually, this might be solved with Ben's pr changes. see if pulling the latest fixes this?

Comment (rocio-desantiago) [reply services/ui-src/src/components/app/AppRoutes.tsx]: Yes, just pulled his changes into the branch and it works now 😄

**Review by BearHanded — APPROVED** [review]

**Review by ailZhou — APPROVED** [review]
lgtm! tested on deploy, admin and stateuser pages are loading and going to the correct dashboard

**Pull request merged**